README.md

```diff
@@ -1,32 +1,505 @@
+
+### Disabling method swizzling
+
+The SDK uses method swizzling by default. Method Swizzling of Push Notification methods means that fewer code changes are required to enable push notifications. Use the following code if you want to disable method swizzling:
+
+```
+- (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions {
+    SwrveConfig* config = [[SwrveConfig alloc] init];
+    config.pushEnabled = YES;
+    config.autoCollectDeviceToken = NO;
+    [[Swrve sharedInstance] initWithAppID:customerAppId apiKey:customerApiKey config:config launchOptions:launchOptions];
+}
+
+- (void)application:(UIApplication *)application didRegisterForRemoteNotificationsWithDeviceToken:(NSData *)deviceToken {
+    if ([Swrve sharedInstance].talk != nil) {
+        [[Swrve sharedInstance].talk setDeviceToken:deviceToken];
+    }
+}
+
+-(void) application:(UIApplication*)application didReceiveRemoteNotification:(NSDictionary *)userInfo {
+    [[Swrve sharedInstance].talk pushNotificationReceived:userInfo];
+}
+```
+
+### Creating and uploading your iOS Certificate
+
+To enable your app to send push notifications to iOS devices, you require a push certificate. A push certificate authorizes an app to receive push notifications and authorizes a service to send push notifications to an app. For more information, see [How Do I Manage iOS Push Certificates?](http://docs.swrve.com/faqs/push-notifications/manage-ios-push-certificates-for-push-notifications/)
+
+### Provisioning your app
+
+Each time your app is published for distribution, you must provision it against the push certificate that has been created for the app. To do this, perform the following steps:
+
+1. In Xcode, on the Product menu, click Archive.
+
+2. On the Organizer, select Archives and then select the latest archive you want to distribute.
+
+3. Select the distribution for ad-hoc deployment.
+
+4. On the provisioning screen, select the development push certificate created for this app.
+
+
+
+Sending Events
+-
+
+### Sending Named Events
+
+```
+[[Swrve sharedInstance] event:@"custom.event_name"];
+```
+
+Rules for sending events:
+
+* Do not send the same named event in differing case.
+
+* Use '.'s in your event name to organize their layout in the Swrve dashboard. Each '.' creates a new tree in the UI which groups your events so they are easy to locate.
+* Do not send more than 1000 unique named events.
+ * Do not add unique identifiers to event names. For example, Tutorial.Start.ServerID-ABDCEFG
+ * Do not add timestamps to event names. For example, Tutorial.Start.1454458885
+* When creating custom events, do not use the `swrve.*` or `Swrve.*` namespace for your own events. This is reserved for Swrve use only. Custom event names beginning with `Swrve.` are restricted and cannot be sent.
+
```

Comment:
Should we add a note that only events sent from the client can be used as triggers for IAM?